UM/Qt/Bindings/OutputDeviceManagerProxy.py

```diff
@@ -68,11 +68,11 @@ def activeDeviceDescription(self) -> str:
     def requestWriteToDevice(self, device_id: str, file_name: str, kwargs: Mapping[str, str]) -> None:
         limit_mimetypes = kwargs.get("limit_mimetypes", None)
         file_type = kwargs.get("file_type", "mesh")
-        preferred_mimetype = kwargs.get("preferred_mimetype", None)
+        preferred_mimetypes = kwargs.get("preferred_mimetypes", None)
```

Comment:
I am worried about using different preferred_mimetype or preferred_mimetypes.
We still have many places where we use "preferred_mimetype"